### DISCUSSION TITLE
Github profile badges and achievements

### DISCUSSION THREAD

**Discussion by crizmo** [community/community#47741] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Issue -
1 - Can see Developer Program Member badge on my profile on mobile device but its not there on github website { PC }
I tried reloading , updating the settings but its still the same

2 - I recently got some of my comments marked as answers on some repos but didnt get the Galaxy Brain achievements 
Tried the same thing on my friends account and he instantly got the badge 

Doubt -
Is something wrong with my settings ?! I am kind of confused 
This happened before too with my github pro badge had to open a ticket to get it fixed 

Should i do the same ?

**Comment by YasinDehfuli** [comment]
Hi You should got two marked answers to get Galaxy Brain achievements If you have two marked answers and you still don't got achievement then you can create ticket for fix it .

It cant help you too : 
https://github.com/Schweinepriester/github-profile-achievements#achievements

**Reply by crizmo** [reply]
Guessed so . I opened a ticket regarding the issue. Ty for the reply

**Comment by github-staff** [comment, marked as answer]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Reply by VerzatileDev** [reply]
I agree with the decision, though I am sad becasue of it as I wish to contribute valuable information in regards to this community. The spam has gone out of control, there are about 20 posts a day regarding the galaxy brain achivement in this community, quite depressing to see these.
